app/src/main/java/com/github/dodobest/upbitapi/UpbitAdapter.kt

```diff
@@ -0,0 +1,52 @@
+package com.github.dodobest.upbitapi
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.github.dodobest.domain.model.UpbitTickerDataWithKoreanName
+import com.github.dodobest.upbitapi.databinding.CoinItemBinding
+import java.text.DecimalFormat
+
+class UpbitAdapter : RecyclerView.Adapter<UpbitViewHolder>() {
+    private val tickerResult: MutableList<UpbitTickerDataWithKoreanName> = mutableListOf()
+
+    private val coinPriceFormat = DecimalFormat("#,###.########")
+    private val changeRateFormat = DecimalFormat("#,###.##%")
+    private val aacTradePriceFormat = DecimalFormat("#,###백만")
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): UpbitViewHolder {
+        val binding = CoinItemBinding.inflate(
+            LayoutInflater.from(parent.context), parent, false
+        )
+        return UpbitViewHolder(binding)
+    }
+
+    override fun onBindViewHolder(holder: UpbitViewHolder, position: Int) {
+        holder.coinMarket.text = tickerResult[position].market
+        holder.coinName.text = tickerResult[position].koreanName
+        holder.coinPrice.text = coinPriceFormat.format(tickerResult[position].tradePrice)
+        holder.changeRate.text =
+            changeRateFormat.format(tickerResult[position].signedChangeRate).toString()
+        holder.aacTradePrice.text =
+            aacTradePriceFormat.format(tickerResult[position].aacTradePrice24h / 1000000)
```

Comment:
```suggestion
            aacTradePriceFormat.format(tickerResult[position].aacTradePrice24h / 1_000_000)
```
천단위에 언더바를 붙이면 가독성이 더 좋아져요.

Reply:
[881e0d6](https://github.com/DoTheBestMayB/UpbitAPI/pull/31/commits/881e0d67e85e7abeb646fe8f709ac81bc4f4f2eb)